web-ui/src/components/reviews/periods/ReviewPeriods.jsx

```diff
@@ -1,162 +1,204 @@
+import MenuItem from '@mui/material/MenuItem';
 import Modal from '@mui/material/Modal';
+import Select from '@mui/material/Select';
 import Skeleton from '@mui/material/Skeleton';
 import TextField from '@mui/material/TextField';
 import Tooltip from '@mui/material/Tooltip';
-import Avatar from '@mui/material/Avatar';
-import WorkIcon from '@mui/icons-material/Work';
-import IconButton from '@mui/material/IconButton';
-import ArchiveIcon from '@mui/icons-material/Archive';
-import UnarchiveIcon from '@mui/icons-material/Unarchive';
-import DeleteIcon from '@mui/icons-material/Delete';
-import Dialog from '@mui/material/Dialog';
-import DialogActions from '@mui/material/DialogActions';
-import DialogContent from '@mui/material/DialogContent';
-import DialogContentText from '@mui/material/DialogContentText';
-import DialogTitle from '@mui/material/DialogTitle';
-import { getReviewPeriods, createReviewPeriod, updateReviewPeriod, removeReviewPeriod } from "../../../api/reviewperiods.js";
-import { findSelfReviewRequestsByPeriodAndTeamMember } from "../../../api/feedback.js";
-import { AppContext } from "../../../context/AppContext";
+import Typography from '@mui/material/Typography';
+
+import {styled} from '@mui/material/styles';
+
+import {findSelfReviewRequestsByPeriodAndTeamMember} from '../../../api/feedback.js';
+import {getAllFeedbackTemplates} from '../../../api/feedbacktemplate.js';
+import {
+  getReviewPeriods,
+  createReviewPeriod,
+  updateReviewPeriod,
+  removeReviewPeriod
+} from '../../../api/reviewperiods.js';
+import {AppContext} from '../../../context/AppContext';
 import {
   UPDATE_REVIEW_PERIODS,
-  ADD_REVIEW_PERIOD,
-} from "../../../context/actions";
+  ADD_REVIEW_PERIOD
+} from '../../../context/actions';
 import {
   selectCsrfToken,
   selectUserProfile,
   selectCurrentUserId,
   selectReviewPeriod,
-  selectReviewPeriods,
-} from "../../../context/selectors";
+  selectReviewPeriods
+} from '../../../context/selectors';
 
 const propTypes = {
   message: PropTypes.string,
-  onSelect: PropTypes.func,
+  onSelect: PropTypes.func
 };
-const displayName = "ReviewPeriods";
+const displayName = 'ReviewPeriods';
+const feedbackTemplateUrl = '/services/feedback/templates';
 
 const PREFIX = displayName;
 const classes = {
   actionButtons: `${PREFIX}-actionButtons`,
   headerContainer: `${PREFIX}-headerContainer`,
-  periodModal: `${PREFIX}-periodModal`,
+  periodModal: `${PREFIX}-periodModal`
 };
 
 const modalStyles = {
-  position: "absolute",
-  minWidth: "400px",
-  maxWidth: "600px",
-  backgroundColor: "background.paper",
-  top: "50%",
-  left: "50%",
-  padding: "1rem",
-  transform: "translate(-50%, -50%)",
-  border: "2px solid #fff",
+  position: 'absolute',
+  minWidth: '400px',
+  maxWidth: '600px',
+  backgroundColor: 'background.paper',
+  top: '50%',
+  left: '50%',
+  padding: '1rem',
+  transform: 'translate(-50%, -50%)',
+  border: '2px solid #fff',
+
+  display: 'flex',
```

Comment:
I added these three CSS properties to get better vertical spacing between the components.

Reply:
Is there a reason for the extra line space that is line 79?

Reply:
I was just highlighting that the three CSS properties I added go together. Removing any one of them loses the effect I'm after.